src/lib/containers/ProgrammaticInstructionsModal.tsx

```diff
@@ -0,0 +1,155 @@
+import React, { useState } from 'react'
```

Comment:
Refactored.  Pulled code from ProgrammaticOptions, which was a modal used in the context of table download.
Then changed ProgrammaticOptions to use this ProgrammaticInstructionsModal, and then renamed ProgrammaticOptions to ProgrammaticTableDownload!